core/embed/rust/src/ui/layout_bolt/mod.rs

```diff
@@ -80,14 +80,7 @@ impl CommonUI for UIBolt {
         show(&mut frame, fade_in);
     }
 
-    fn screen_update() {
-        let mut frame = ErrorScreen::new(
-            "Update".into(),
-            "Finishing firmware update".into(),
-            "Do not turn off your Trezor".into(),
-        );
-        show(&mut frame, true);
-    }
+    fn screen_update() {}
```

Comment:
making these functions empty looks wrong at first glance
should we drop the whole function, if it is unused (as it seems, from the fact that Caesar / Delicia don't even have it)?

Reply:
All UIs have it and it is called (when bootloader is updating). We might want to implement it with some nice message, but since the update us fast i didnt want to waste my time with thay right now. On Bolt and Delizia this will cause quick fadeout-fadein when updating the bootloader,  no effect on Ceasar. Cc also @Hannsek if this is acceptable.